docs/kubernetes.md

```diff
@@ -0,0 +1,8 @@
+# Kubernetes Concepts for Gluster Storage
+
+[Persistent Volumes] (http://kubernetes.io/docs/user-guide/persistent-volumes/)
```

Comment:
When you view this it looks to be in same line. can we be able to make it in new line? would be more readable.

View : https://github.com/ramkrsna/gluster-kubernetes/blob/addDocs/docs/kubernetes.md